### PR TITLE
Update a.js

### PR DESCRIPTION
**Pull request by Victorayo**
In reference to PR #157

### CODE REVIEW

File: src/techCompanies/orderByAlphabeth/a.js [hunk 1]
```diff
@@ -22,6 +22,13 @@ export default {
       industry: "Logistics",
       website: "https://www.ace.ng",
     },
+    {
+      companyName: "Accelerex)",
```
Comment (gray-adeyi) [inline src/techCompanies/orderByAlphabeth/a.js]: You have a misplaced closing parenthesis. Fix it

File: src/techCompanies/orderByAlphabeth/a.js [hunk 2]
```diff
@@ -22,6 +22,13 @@ export default {
...
+      companyTwitterHandle: "accelerexhq",
+      foundersTwitterHandle: [" "],
```
Comment (gray-adeyi) [inline src/techCompanies/orderByAlphabeth/a.js]: Just leave the array empty `foundersTwitterHandle: []`

**Review by gray-adeyi — CHANGES_REQUESTED** [review]

File: src/techCompanies/orderByAlphabeth/a.js (hunk 1 above)
Comment (Victorayo) [reply src/techCompanies/orderByAlphabeth/a.js]: Alright. Will do that straightaway

File: src/techCompanies/orderByAlphabeth/a.js (hunk 2 above)
Comment (Victorayo) [reply src/techCompanies/orderByAlphabeth/a.js]: Alright, will do that straightaway

**Comment by Victorayo** [comment]
The changes you requested has been made @gray-adeyi

**Pull request merged**